functions.php

```diff
@@ -49,4 +49,49 @@ function createTable(array $dbQuery) {
     return $table;
 }
 
+/**
+ * Checks length of string is less than 100 and return boolean
+ *
+ * @param string $entry string to be tested
+ * @return boolean true if length is less than 100, else false.
+ */
+function strLength(string $entry) :bool {
+    if(strlen($entry) < 100 && strlen($entry) !== 0 && isset($entry)) {
```

Comment:
Whenever you're doing a comparison like this, and then wanting to return true/false as a result, you can simplify the code like this:

`return strlen($entry) < 100 && strlen($entry) !== 0`

Remember that all comparisons evaluate to a boolean, so if the comparison is true, the function returns true, and false if not.